content/en/docs/concepts/sampling/index.md

```diff
@@ -45,6 +32,70 @@ Sometimes, the definitions of these terms get mixed up. You might find someone
 states that they are "sampling out data" or that data not processed or exported
 is considered "sampled". These are incorrect statements.
 
+## Why Sample?
+
+Sampling is one of the most effective ways to reduce the costs of observability
+without losing visbility. Although there are other ways to lower costs, such as
+filtering or aggregating data, these other methods do not adhere to the concept
+of _representativeness_, which is crucial when performing in-depth analysis of
+application or system behavior.
+
+Representativeness is the principle that a smaller group can accurately
+represent a larger group. Additionally, representativeness can be mathematically
+verified, meaning that you can have high confidence that a smaller sample of
+data accurately represents the larger group.
+
+Additionally, the more data you generate, the less data you actually need to
+have a representative sample. For high-volume systems, is quite common for a
+sampling rate of 1% or lower to very accurately represent the other 99% of data.
+
+### When to Sample
+
+You should consider sampling if you meet any of the following criteria:
+
+- You generate 1000 or more traces per second
+- Most of your trace data represents healthy traffic with little variation in
+  data
+- You have some common criteria, like errors or high latency, that usually means
+  something is wrong
+- You have domain-specific criteria you can use to determine relevant data
+  beyond errors and latency
+- You can describe some common rules that determine if data should be sampled or
+  dropped
+- You have a tell your services apart, so that high- and low-volume services are
+  sampled differently
+- You have the ability to route unsampled data (for "just in case" scenarios) to
+  low-cost storage systems
+
+Finally, consider your overall budget. If you have limited budget for
+Observability, but can afford to spend time to effectively sample, then sampling
+can generally be worth it.
+
+### When not to Sample
+
+Sampling may not be appropriate for you. You may want to avoid sampling if you
+meet any of the following criteria:
```

Comment:
```suggestion
Sampling may not be appropriate for you. You may want to avoid sampling if you
meet any of the following criteria:
```
```suggestion
Sampling may not be appropriate for you. You may want to avoid sampling if you
meet any of the following criteria:
```
```suggestion
Sampling might not be appropriate for you. You might want to avoid sampling if
you meet any of the following criteria:
```